src/eva/vision/models/networks/backbones/pathology/histai.py

```diff
@@ -42,5 +42,5 @@ def histai_hibou_l(out_indices: int | Tuple[int, ...] | None = None) -> nn.Modul
         model_name="histai/hibou-L",
         out_indices=out_indices,
         model_kwargs={"trust_remote_code": True},
-        transform_args={"ignore_remaining_dims": True} if out_indices is not None else None,
+        transform_args={"num_reg_tokens": 4} if out_indices is not None else None,
```

Comment:
maybe make this a variable, or explain in a comment where "4" is coming from?

Reply:
This needs to be a constant, is a hyperparameter of the ViT (their models use 4 register tokens).

Reply:
Added a comment now to the docstring.